src/components/screen/Calender/Calender.stories.tsx

```diff
@@ -0,0 +1,13 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import Calender from './Calender';
+
+const meta: Meta<typeof Calender> = {
+  title: 'common/Calender',
```

Comment:
ここってStorybookで表示される名前を決めてるんだけど、Calenderはscreenの中にあるから、名前もそれに合わせて方がいいかな！
```suggestion
  title: 'screen/Calender',
```

Reply:
修正した
https://github.com/NUTFes/chosei-chan/pull/22/commits/8161d8880e085321dec4babca8ce8ec752513b1f